src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -51,27 +55,38 @@
     private static final Logger log = LoggerFactory.getLogger(Application.class);
 
     private static final String EMBEDDED_HTTP_SERVER_PORT = "EMBEDDED_HTTP_SERVER_PORT";
+    private static final String BRIDGE_METRICS_ENABLED = "BRIDGE_METRICS_ENABLED";
 
     private static final int DEFAULT_EMBEDDED_HTTP_SERVER_PORT = 8080;
 
+
+
```

Comment:
Do we need 3 empty lines? Isn't that a bit too much? Maybe two would be sufficient :-P

Reply:
even just one!